node_helper.js

```diff
@@ -15,15 +15,34 @@ module.exports = NodeHelper.create({
 		if(notification === "GET_COMIC"){
 			
 			var comicJsonUri = payload.config.dailyJsonUrl;
+			var comic;
+			var rndcomic;
+			var rndUrl;
+			var body;
+			var d = new Date();
+			var n = d.getDay(); 
 			
 			request(comicJsonUri, function (error, response, body) {
 				if (!error && response.statusCode == 200) {
 					console.log(body);
-					self.sendSocketNotification("COMIC", JSON.parse(body));
-					console.log(JSON.parse(body).img);
-					
+					if ( this.n == 1 || this.n == 3 || this.n == 5 ) {
```

Comment:
Have you tested this? Since this is executed within a callback function, I don't think this.n is referring to the local var n declared earlier. Same for the other references to "this" later in the function body.